lib/iris/tests/unit/experimental/fieldsfile/test__convert_collation.py

```diff
@@ -1,325 +0,0 @@
-# (C) British Crown Copyright 2014 - 2015, Met Office
```

Comment:
We seem to have missed a trick here.
This function still exists, in a slightly enhanced form, as `iris.fileformats.um._fast_load.convert_collation`
So this could usefully be re-purposed to that, rather than discarded
I.E. --> `iris.tests.unit.fileformats.um.fast_load.test__convert_collation.py`

I checked + it works unmodified, just change the import "... as convert_collation".